### ISSUE TITLE
Question: Support for the Dialing UI on Navigation Screens

### ISSUE THREAD

**Issue by multilan-tarek** [tedsalmon/BlueBus#126]
Hello!

Quick question: Is support for the dialing ui on navigation screens planned?

Greeting from Germany!

**Comment by tedsalmon** [comment]
Tarek,

@nabucho has been working on this feverishly in branch `feature/phone`

**Comment by multilan-tarek** [comment]
Great work, that's really cool!

Would it be possible to display messages in the phone menu using the Bluetooth MAP profile? Not that it's really useful, but it would be a cool gimmick.

**Comment by tedsalmon** [comment]
> Great work, that's really cool!
> 
> Would it be possible to display messages in the phone menu using the Bluetooth MAP profile? Not that it's really useful, but it would be a cool gimmick.

The BM83 doesn't support MAP, but the BC127 does. Because of this disparity, we will not be supporting SMS readouts on the Nav UI. :(

**Comment by Keff789** [comment]
SMS is anyways not useful, if something push notifications like Whatsapp would be cool :p but that seems way out of scope

**Comment by nabucho** [comment]
Yes, general notifications (ANP profile) are unfortunately not at all supported by the Bluetooth chips used in BlueBus.
